lib/Client.js

```diff
@@ -1958,13 +1969,18 @@ class Client extends EventEmitter {
     * @arg {Boolean} [options.allowedMentions.repliedUser] Whether or not to mention the author of the message being replied to.
     * @arg {Boolean | Array<String>} [options.allowedMentions.roles] Whether or not to allow all role mentions, or an array of specific role mentions to allow.
     * @arg {Boolean | Array<String>} [options.allowedMentions.users] Whether or not to allow all user mentions, or an array of specific user mentions to allow.
+    * @arg {Boolean} [options.allowedMentions.repliedUser] Whether or not to mention the author of the message being replied to.
+    * @arg {Array<Object>} [options.attachments] Attachment objects to retain and/or edit the name and/or description for currently attached files and add the filename and description for new files
+    * @arg {String} [options.attachments[].description] The file description
+    * @arg {String} [options.attachments[].filename] The file's name. This is not needed if you are attaching a new file
+    * @arg {Number} options.attachments[].id The file's ID. If you are attaching a new file, this would be the index ID of the attaching file
```

Comment:
```suggestion
    * @arg {(Number | String)} options.attachments[].id The file's ID. If you are attaching a new file, this would be the index ID of the attaching file
```